5_databases/week2/part1.sql

```diff
@@ -0,0 +1,26 @@
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id) VALUES ("Become a developer", "Work hard", "2017-10-06 04:03:52", "2017-09-20 19:34:43", "2017-12-22 20:58:03", 2)
+
+--Change the title of a task
+UPDATE task
+SET title = "go to hyf"
+WHERE title = "Buy gift for Paul";
```

Comment:
Correct, but better to do with a primary key (id) so you dont edit more than one by accident